requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy>=1.7.0
+scipy>=1.7.0
```

Comment:
`ml_datasets` is a dependency of `spacy`, so I want to be mindful of pulling in additional third-party dependencies here.

In fact, our [contributing guide](https://github.com/explosion/spaCy/blob/master/CONTRIBUTING.md) says:
>  Python has a very rich ecosystem. Libraries like PyTorch, TensorFlow, scikit-learn, SciPy or Gensim do lots of useful things — but we don't want to have them as default dependencies.

Is there a way we can avoid the `loadmat` functionality?